easybuild/easyconfigs/g/gimkl/gimkl-2017a.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = "Toolchain"
+
+name = 'gimkl'
+version = '2017a'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+comp = ('GCC', '5.4.0')
```

Comment:
If you reintroduce the `-2.26` in the versionsuffix, you can make this `5.4.0-2.26` (which is not 100% correct strictly speaking, but it's unlikely to cause problems).